WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -680,10 +680,10 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
         // Now that we have the image, create an aspect ratio constraint for
         // the featuredImageView
         let ratio = size.height / size.width
-        let constraint = NSLayoutConstraint(item: featuredImageView,
+        let constraint = NSLayoutConstraint(item: featuredImageView!,
```

Comment:
I don't understand these new force unwraps and don't see anything relevant in the [migration guide](https://swift.org/migration-guide-swift5/). `featuredImageView` is already defined as `CachedAnimatedImageView!`

Reply:
I was also surprised by this one... if you remove the ! you will see a warning showing up.